bob-api/index.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+// Include the api
```

Comment:
onnodige comment?

Reply:
Waarom niet? Misschien vind ik wel dat die nodig is.

Je benoemt nu een feit, probeer het de volgende keer meer vanuit jezelf te verwoorden. Iets van: 'Mij lijkt deze comment niet perse nodig' of 'kan je mij vertellen wat deze comment toevoegt'?